static/app/views/organizationContext.tsx

```diff
@@ -56,12 +57,13 @@ type State = {
   };
 };
 
-class OrganizationContext extends React.Component<Props, State> {
+export const OrganizationContext = createContext<Organization | null>(null);
```

Comment:
Can't this also be a LightWeightOrganization sometimes?

Reply:
should we export the context below together with the other exports?

Reply:
I was following the typing coming out of `organizationStore`, I thought only Organization would be getting set? Where does this state get modified and `LightweightOrganization` set?

Does that mean it's not safe to access `projects` and `teams` coming from the org store?